src/Nox.Ui.Blazor.Lib/Components/NoxTypes/EditMonth.razor.cs

```diff
@@ -0,0 +1,68 @@
+﻿using Microsoft.AspNetCore.Components;
+using Nox.Types;
+using Nox.Ui.Blazor.Lib.Models;
+using System.Globalization;
+
+namespace Nox.Ui.Blazor.Lib.Components.NoxTypes;
+
+public partial class EditMonth : ComponentBase
+{
+
+    #region Declarations
+
+    [Parameter]
+    public byte? Month { get; set; }
+
+    [Parameter]
+    public CultureInfo CultureInfo { get; set; } = CultureInfo.CurrentCulture;
+
+    [Parameter]
+    public string? Title { get; set; }
+
+    [Parameter]
+    public EventCallback<byte?> MonthChanged { get; set; }
+
+    public Dictionary<byte, string?>? MonthSelectionList { get; set; } = new Dictionary<byte, string?>();
+
+    public string? CurrentMonthStr { get; set; }
+
+    #endregion
+
+    /// <summary>
+    /// Handles initial loading
+    /// </summary>
+    protected override void OnInitialized()
+    {
```

Comment:
can we make MonthSelectionList static and fill up on the static constructor?

This way will be share by all instances and improves memory and performance.

Reply:
made MonthSelectionList static but dont think I can create the static constructor because it uses CultureInfo param which has to be non-static ? - have created it on EditTime okay though